starters/solidstart-tanstackquery-tailwind-modules/src/components/CounterExample/Counter.tsx

```diff
@@ -0,0 +1,43 @@
+import { createSignal } from "solid-js";
+import { ActionButtonsData, CounterData } from "./types";
+
+const getCounterData = (): CounterData => {
+  const [count, setCount] = createSignal<number>(0);
```

Comment:
- The counter should be controlled by the global state manager
As it stands it is not global. You can check the other solidjs project to see how we did it. But I think you could just move this outside the function.